pom.xml

```diff
@@ -42,30 +42,378 @@
-    <kettle.version>6.0-SNAPSHOT</kettle.version>
-    <notification.version>6.0-SNAPSHOT</notification.version>
+    <kettle.version>${project.version}</kettle.version>
+    <notification.version>${project.version}</notification.version>
+    <pentaho-i18n-bundle.version>${project.version}</pentaho-i18n-bundle.version>
     <bean-matchers.version>0.9</bean-matchers.version>
+ 	
+    <source.jdk.version>1.6</source.jdk.version>
+    <target.jdk.version>1.6</target.jdk.version>
+ 	
+ 	<plugin.org.apache.maven.plugins.maven-resources-plugin.version>2.7</plugin.org.apache.maven.plugins.maven-resources-plugin.version>
+    <plugin.org.apache.maven.plugins.maven-compiler-plugin.version>3.0</plugin.org.apache.maven.plugins.maven-compiler-plugin.version> 
+    <plugin.org.apache.maven.plugins.maven-javadoc-plugin.version>2.10.2</plugin.org.apache.maven.plugins.maven-javadoc-plugin.version>
+    <plugin.org.apache.maven.plugins.maven-jar-plugin.version>2.5</plugin.org.apache.maven.plugins.maven-jar-plugin.version>
+    <plugin.org.apache.felix.maven-bundle-plugin.version>2.3.7</plugin.org.apache.felix.maven-bundle-plugin.version>
+    <plugin.org.codehaus.mojo.build-helper-maven-plugin.version>1.5</plugin.org.codehaus.mojo.build-helper-maven-plugin.version>
+    <plugin.org.apache.karaf.tooling.karaf.karaf-maven-plugin.version>${karaf.version}</plugin.org.apache.karaf.tooling.karaf.karaf-maven-plugin.version>
+	<plugin.com.github.klieber.phantomjs-maven-plugin.version>0.4</plugin.com.github.klieber.phantomjs-maven-plugin.version>
+    <plugin.com.github.searls.jasmine-maven-plugin.version>1.3.1.5</plugin.com.github.searls.jasmine-maven-plugin.version>
+    <plugin.com.github.timurstrekalov.saga-maven-plugin.version>1.5.3</plugin.com.github.timurstrekalov.saga-maven-plugin.version>
   </properties>
 
-  <scm>
-    <connection>scm:git:git@github.com:${github.user}/${project.artifactId}.git</connection>
-    <url>scm:git:git@github.com:${github.user}/${project.artifactId}.git</url>
-    <developerConnection>scm:git:git@github.com:${github.user}/${project.artifactId}.git</developerConnection>
-    <tag>HEAD</tag>
-  </scm>
+  
+  <!-- copied from documentation pom.xml -->
+   <profiles>
+    <profile>
+      <id>default-profile</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+        <file>
+          <exists>${java.home}/../lib/tools.jar</exists>
+        </file>
+      </activation>
+      <properties>
+        <toolsjar>${java.home}/../lib/tools.jar</toolsjar>
+      </properties>
+    </profile>
+    <profile>
+      <id>mac-profile</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+        <file>
+          <exists>${java.home}/../Classes/classes.jar</exists>
+        </file>
+      </activation>
+      <properties>
+        <toolsjar>${java.home}/../Classes/classes.jar</toolsjar>
+      </properties>
+    </profile>
+  </profiles>
+  
+<dependencyManagement>
+  <dependencies>
```

Comment:
Validate that dependencies are actually used in child POMs. Remove them if they are not used.
If a dependency is only used by one of the child POMs then remove it from here and add it to that child POM.